src/index.js

```diff
@@ -1,8 +1,12 @@
-var _       = require("lodash"),
+const _assign = require("lodash.assign"),
```

Comment:
I'd rather not use es6 here to remain as compatible as possible with other frameworks. Also, I'd prefer the ordering of the requires to be alphabetical.